src/components/button/index.jsx

```diff
@@ -20,7 +20,7 @@ function Button({ children, href, type, className: otherClass }) {
         {children}
       </Link>
     ) : (
-      <a href={href} className={className}>
+      <a href={href} className={className} {...props}>
```

Comment:
Why not add `target="_blank" rel="noopener noreferrer"` directly here?

Reply:
Because we have one link going to https://tech4covid19.org, which should be opened on a new tab and also other app-related links like https://covidografia.pt/app which should open on the same tab being of the same scope. Hope this makes sense 🙂

Reply:
We can probably refactor to check if the link passed contains the `siteUrl` as prefix and if not automatically add the `target="_blank" rel="noopener noreferrer"`

Reply:
I can push a commit with this last approach if we agree it's a good solution.

Reply:
Actually i agree with @taniaprocha 
Because we're already checking if the link is internal or not. So, for external urls, i think that makes sense to add it here. 🙂

Reply:
I pushed a commit that adds the `target="_blank" rel="noopener noreferrer"` dynamically by checking the link domain 🙂

Reply:
Check URL inside `Button` component doesn't seem a good idea to me. It would be better if we have a boolean prop to new tab or not, what do you think?

Reply:
With the boolean prop, we would need to know the link beforehand and then add the prop accordingly. The URL check approach scales with the project configuration. If in the future we want to change this behavior, we only need to change in one place 🙂 .

Reply:
Ok 👍